spec/features/staff/teammates_spec.rb

```diff
@@ -88,4 +88,35 @@
     end
   end
 
+  context "Organizer can't send invitations if contact email for event isn't set" do
+    it "redirects them to the edit page and then back to the teammates page", js: true do
+      incomplete_event = create(:event, name: "My Event", contact_email: "")
+      create(:teammate, event: incomplete_event, user: organizer_user, role: "organizer")
+
+      login_as(organizer_user)
+      visit event_staff_teammates_path(incomplete_event)
+
+      click_link "Invite new teammate"
+      fill_in "Email", with: "harrypotter@hogwarts.edu"
+      select("reviewer", from: "Role")
+      click_button "Invite"
+
+      expect(current_path).to eq(event_staff_edit_path(incomplete_event))
+      expect(page).to have_content "You must set a contact email for this event before inviting teammates."
+      fill_in "event[contact_email]", with: "incomplete@email.com"
```

Comment:
I think everything up through asserting the flash message is sufficient. Since the rest of this test is re-testing functionality that's already been verified earlier in the suite, you can leave it out.